backend/src/db/mongoose.js

```diff
@@ -1,7 +1,9 @@
 const mongoose = require('mongoose');
 
-// ONLY CALL THIS WHEN COURSES COLLECTION IS INITIALIZED
-const { duplicateNecessaryCourses } = require('../scripts/duplicate-courses');
+// ONLY RUN THIS WHEN COURSES COLLECTION IS INITIALIZED
+const { duplicateNecessaryCourses } = require('../scripts/db-scripts');
```

Comment:
const { duplicateNecessaryCourses, removeFieldFromCourses } = require('../scripts/db-scripts') should work too

Reply:
@anthopark

Reply:
@emmohac I would like explicit comments for each imported function. These functions will not be called often as it manipulates DB. I would prefer to leave this way.

Reply:
@anthopark When using eslint (highly suggest eslint + prettier) in the codebase, it will complain about this duplication import statement and suggest a 'quick fix'.
So it can be done this way if the comments are needed
```
const {
// ONLY RUN THIS WHEN COURSES COLLECTION IS INITIALIZED
duplicateNecessaryCourses, 
// RUN AFTER UPDATING PREVQUARTERS ARE UPDATED, TO REMOVE "prevQuarter" FROM RESEARCH CREDIT COURSES
removeFieldFromCourses,
} = require('../scripts/db-scripts')
```
What do you think?

Reply:
@emmohac Haven't thought of this way. I think it's better. I'll make the change. btw are u interested in including ESlint in this project? I see the benefit of it and was wondering if you'd like to configure it.

Reply:
@anthopark yeah. I used ESLint airbnb extension. I can help setting it up too. But let's make the change for this PR and then I'll try adding eslint and prettier.